pkg/tsdb/cloudmonitoring/resource_handler_test.go

```diff
@@ -285,3 +287,23 @@ func Test_processData_functions(t *testing.T) {
 		})
 	}
 }
+
+func Test_getGCEDefaultProject(t *testing.T) {
+	project := "test-project"
+	s := Service{
+		im: &fakeInstance{
+			services: map[string]datasourceService{
+				cloudMonitor: {
+					url:    routes[cloudMonitor].url,
+					client: &http.Client{},
+				},
+			},
+		},
+		gceDefaultProjectGetter: func(ctx context.Context) (string, error) {
+			return project, nil
+		},
+	}
+
+	assert.HTTPSuccess(t, s.getGCEDefaultProject(), "GET", "/gceDefaultProject", nil)
+	assert.HTTPBodyContains(t, s.getGCEDefaultProject(), "GET", "/gceDefaultProject", nil, project)
```

Comment:
I believe this test would have successed without your change. I am not sure if the fix is that since now you are returning a JSON, there is a header `Content-Type: "application/json"` or that we are now returning `"project-name"` vs `project-name`. Can you clarify that?

Reply:
Previously the return value would've been `project-name` which is invalid JSON vs `"project-name"`. You're correct the test would've succeeded before the change, I'm updating this now 😊